openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/3-functional/VQDN-Backup-Restore-CSI/backup-restore-csi

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
```

Comment:
@shashank855 In cstor-volume-resize (ABZF) task in function stage we are using the busybox application deployed in the `deploy-busybox-csi` task which you replaced with backup restore on percona application..!!!

Reply:
Yes. This way we will test application provisioning using CSI along with backup and restore.